services/nyx/launch-worker.sh

```diff
@@ -204,9 +204,20 @@ else
   if [[ -n "$TASK_ID" ]] || [[ -n "$RUN_ID" ]]; then
     DAEMON_ARGS+=(--afl-hide-logs)
   fi
-  # Download the corpus from S3
-  update-status "downloading corpus"
-  time guided-fuzzing-daemon "${S3_PROJECT_ARGS[@]}" --s3-corpus-download ./corpus
+
+  # Sometimes, don't download the existing corpus.
+  # This can increase coverage in large targets and prevents bad corpora.
+  # Results will be merged with the existing corpus on next refresh.
+  if [[ $(python3 -c "import random;print(random.randint(1,100))") -lt 98 ]]; then
```

Comment:
I think `-gt 1` is less of a foot.

Reply:
changed to `-le` to fix the off-by-one ([1, 100] <= 98)